### PR TITLE
use status_info not uri module in inventory_plugin test

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: tests/test_playbooks/inventory_plugin.yml [hunk 1]
```diff
@@ -52,14 +52,12 @@
...
-        url_password: changeme
-        force_basic_auth: true
-        status_code: 200
+      status_info:
+        username: admin
+        password: changeme
+        server_url: http://127.0.0.1:3000/
       register: result
-      until: result.status == 200
+      until: result.status is defined and result.status.status == 200
```
Comment (mdellweg) [inline tests/test_playbooks/inventory_plugin.yml]: Not tested, but i think cleaner:
```suggestion
      until:
        - result is success
        - result.status.status == 200
```

Comment (evgeni) [reply tests/test_playbooks/inventory_plugin.yml]: One way to find out…

**Comment by evgeni** [comment]
Seems to work (ignore the failed ones, galaxy was acting up)

**Comment by evgeni** [comment]
:green_apple:

**Review by mdellweg — APPROVED** [review]

**Pull request merged**